USERS.md

```diff
@@ -405,4 +405,5 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [Zimpler](https://www.zimpler.com/)
 1. [ZipRecuiter](https://www.ziprecruiter.com/)
 1. [ZOZO](https://corp.zozo.com/)
+1. [T-ROC Global](https://trocglobal.com/)
```

Comment:
Can you please add it in the alphabetically sorted order like others?

Reply:
+1 to Ishita's comment. I think it should come at line 363.